Makefile

```diff
@@ -16,10 +18,24 @@ cook-image:
 push-image:
 	docker push $(DOCKER_TAG)
 
-.PHONY: test
-test:
-	tox
+venv: requirements.txt
+	python ./vendor/venv-update venv= venv -ppython3 install= -r requirements.txt
+
+.PHONY: install-hooks
+install-hooks: venv
+	$(BIN)/pre-commit install -f --install-hooks
+
+test: lint
+
+.PHONY: lint
+lint: venv
+	$(BIN)/pre-commit run --all-files
 
 .PHONY: clean
 clean:
-	rm -r .tox
+	rm -rf venv
+
+.PHONY: update-requirements
+update-requirements: venv
+	$(BIN)/upgrade-requirements
```

Comment:
The way `upgrade-requirements` works (there's a more in-depth description [here](https://github.com/Yelp/requirements-tools#recommended-requirements-setup-for-applications)), you'll need to have a `requirements-minimal.txt` too, and you probably want it to be `requirements-dev-minimal.txt` actually instead since we only have dev requirements here, nothing that really is used in the actual built image.

Reply:
Looks like I will still need an empty `requirements.txt` here for `upgrade-requirements` to work.